components/Cards/README.md

```diff
@@ -58,6 +109,10 @@ pod install
 
 ## Usage
 
+An `MDCCard` can be added and used as you would add any `UIView` or `UIControl`, if manually in code, or through the interface builder.
+
+An `MDCCardCollectionCell` can be added, used, and reused as a `UICollectionViewCell`, if manually in code, or through the interface builder.
+
 ### Importing
 
 Before using Cards, you'll need to import it:
```

Comment:
<!--<div class="material-code-render" markdown="1">-->
#### Swift
``` swift
import MaterialComponents.MaterialCards
```


#### Objective-C
``` objc
#import "MaterialCards.h"
```
<!--</div>-->

Reply:
I'm not sure what to do here; this looks like a copy of what we already have?

Reply:
I don't see the actual import code sample in the existing PR.
The Swift & ObjC sample code would be consistent with our other readmes:
See Usage - Importing in:
https://github.com/material-components/material-components-ios/blob/develop/components/FeatureHighlight/README.md

Reply:
It was collapsed.  Disregard.